src/registry.rs

```diff
@@ -57,7 +57,6 @@ use std::borrow::Cow;
 /// #                "# EOF\n";
 /// # assert_eq!(expected, String::from_utf8(buffer).unwrap());
 /// ```
-#[derive(Debug)]
```

Comment:
Why this change?

Reply:
Sorry, it was because of unsuccessful conflicts with rebase, fixed